kebechet/managers/thoth_advise/thoth_advise.py

```diff
@@ -311,51 +311,43 @@ def run(self, labels: list, analysis_id=None):
 
                 with open(".thoth.yaml", "r") as f:
                     thoth_config = yaml.safe_load(f)
-
-                if thoth_config.get("overlays_dir"):
-                    runtime_environments = [
```

Comment:
As we are removing the local `runtime_environments` var, let's also remove its declaration above (line 297)

Reply:
ack,done